scripts/aci-tile-config.py

```diff
@@ -6,13 +6,19 @@
 import requests
 import sys
 import yaml
+import warnings
 
-import urllib3
-urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+if sys.version_info[0] >= 3:
+   import urllib3
+   urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
Another option would have been to just put this into a try/except block (and not have to even bother about the python version).